src/renderers/WebGLMultisampleRenderTarget.js

```diff
@@ -0,0 +1,35 @@
+import { WebGLRenderTarget } from './WebGLRenderTarget.js';
+
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ * @author Matt DesLauriers / @mattdesl
+ */
+
+function WebGLMultisampleRenderTarget( width, height, options ) {
+
+	WebGLRenderTarget.call( this, width, height, options );
+
+	this.samples = 4;
```

Comment:
Should this be settable via the options parameter??

```js
new WebGLMultisampleRenderTarget( 256, 256, { samples: 4 });
```

Reply:
Absolutely! 👍

Reply:
I think we should stop doing this type of API.
Defining `this.samples = 4;` in the constructor should be good for now.

Reply:
Okay, I revert the change.